SIL.Core.Desktop.Tests/UsbDrive/UsbDeviceInfoTests.cs

```diff
@@ -7,7 +7,9 @@
 using NUnit.Framework;
 using SIL.PlatformUtilities;
 using SIL.UsbDrive;
+#if !NET
 using SIL.UsbDrive.Linux;
+#endif
```

Comment:
let's remove this using statement, and just access `UsbDriveInfoUDisks2` on line 191 via it's full name